coderd/database/migrations/000306_update_protect_deleting_organization_function.up.sql

```diff
@@ -0,0 +1,96 @@
+DROP TRIGGER IF EXISTS protect_deleting_organizations ON organizations;
```

Comment:
seems a bit over defensive, but also doesn't really hurt anything.

mostly I'm just wondering if you think this needs to stay or if this is just some weird thing that your little buddy dreamt up 😂